src/Sulu/Bundle/CustomUrlBundle/Controller/CustomUrlController.php

```diff
@@ -95,7 +95,7 @@ public function postAction($webspaceKey, Request $request)
         $document = $this->get('sulu_custom_urls.manager')->create(
             $webspaceKey,
             $request->request->all(),
-            $this->getLocale($request)
+            $this->getRequestParameter($request, 'targetLocale', true)
```

Comment:
The `getLocale` method should actually return null, and at the same time the `locale` must be a mandatory parameter for the `cgetAction` I'd say. Then everybody can access the page without weird locale permission errors, and at the same time the title for the list would be correct.